rotkehlchen/api/v1/schemas.py

```diff
@@ -3130,3 +3130,55 @@ class ClearAvatarsCacheSchema(Schema):
         fields.String(required=True, validate=lambda x: x.endswith('.eth')),
         load_default=None,
     )
+
+
+class EthStakingHistoryStatsProfit(Schema):
+    """Schema for querying ethereum staking history stats"""
+    from_timestamp = TimestampField(load_default=Timestamp(0))
+    to_timestamp = TimestampField(load_default=ts_now)
+    addresses = DelimitedOrNormalList(EvmAddressField, load_default=None)
+    validator_indices = DelimitedOrNormalList(fields.Integer(), load_default=None)
+
+    def __init__(self, chains_aggregator: 'ChainsAggregator') -> None:
+        super().__init__()
+        self.chains_aggregator = chains_aggregator
+
+    @post_load
+    def make_history_event_filter(
+            self,
+            data: dict[str, Any],
+            **_kwargs: Any,
+    ) -> dict[str, Any]:
+        if (addresses := data['addresses']) is None:
+            addresses = self.chains_aggregator.queried_addresses_for_module('eth2')
+
+        common_arguments = {
+            'from_ts': data['from_timestamp'],
+            'to_ts': data['to_timestamp'],
+            'location_labels': addresses,
+            'validator_indices': data['validator_indices'],
+        }
+
+        withdrawals_filter_query = EthStakingEventFilterQuery.make(
+            **common_arguments,
+            event_types=[HistoryEventType.STAKING],
+            event_subtypes=[HistoryEventSubType.REMOVE_ASSET],
+            entry_types=[HistoryBaseEntryType.ETH_WITHDRAWAL_EVENT],
+        )
+        execution_filter_query = EthStakingEventFilterQuery.make(
+            **common_arguments,
+            event_types=[HistoryEventType.STAKING],
+            event_subtypes=[HistoryEventSubType.BLOCK_PRODUCTION, HistoryEventSubType.MEV_REWARD],
+            entry_types=[HistoryBaseEntryType.ETH_BLOCK_EVENT],
+        )
+
+        return {
+            'withdrawals_filter_query': withdrawals_filter_query,
+            'execution_filter_query': execution_filter_query,
+        }
+
+
+class EthStakingHistoryStatsDetails(AsyncIgnoreCacheQueryArgumentSchema):
+    """Schema for querying ethereum staking history details"""
+    location_labels = DelimitedOrNormalList(EvmAddressField, load_default=None)
+    validator_indices = DelimitedOrNormalList(fields.Integer(), load_default=None)
```

Comment:
Probably nitpick but you could abstract the 

```python
    location_labels = DelimitedOrNormalList(EvmAddressField, load_default=None)
    validator_indices = DelimitedOrNormalList(fields.Integer(), load_default=None)
```

to its own schema interface and add it to both `EthStakingHistoryStatsDetails` and `EthStakingHistoryStatsProfit`